src/box/raft.c

```diff
@@ -435,6 +438,65 @@ box_raft_wait_term_persisted(void)
 	return 0;
 }
 
+static bool
+box_raft_term_broadcasted(struct raft *raft)
+{
+	replicaset_foreach(replica) {
+		/*
+		 * Skip self; anonymous replicas and replicas without relay
+		 * (assume they will recieve all broadcasts from our WAL
+		 * in correct order, when they connect).
+		 */
+		if (replica->relay == NULL || replica->anon ||
+				tt_uuid_is_equal(&replica->uuid, &INSTANCE_UUID))
```

Comment:
Lets fix the indentation. Multiline `if`s need to be aligned by the opening brace + 1:
```C
if (replica->relay == NULL || replica->anon ||
    tt_uuid_is_equal(&replica->uuid, &INSTANCE_UUID))
```
The same below.